packages/web-components/src/kumo/tablist/tablist.ts

```diff
@@ -0,0 +1,7 @@
+import { BaseTablist } from '../../tablist/tablist.js';
+
+/**
+ * Kumo tablist.
+ * @public
+ */
+export class KumoTablist extends BaseTablist {}
```

Comment:
Is this needed or are you including it because you know you will need to add stuff later?

Reply:
I assume it'll need something later. Or perhaps two different tab components. Unclear. How about I rip it out for now.